policies/tickets.rego

```diff
@@ -0,0 +1,24 @@
+package tickets
+
+import rego.v1
+
+roles_in_tenant := {
+    "acmecorp": {
+      "alice": ["admin"],
+      "bob": ["reader"],
+      "ceasar": ["reader", "resolver"],
+    },
+    "hooli": {
+      "dylan": ["admin"],
+    }
+}
+
+default allow := false
```

Comment:
I have been finding it useful for debugging to have a policy entrypoint that echos back whatever `input` was.